common/country_formation/00_major_formables.txt

```diff
@@ -0,0 +1,154 @@
+﻿GER = {
+	use_culture_states = yes
+	
+	is_major_formation = yes
+	
+	unification_play = dp_unify_germany
+	leadership_play = dp_leadership_germany
+
+	required_states_fraction = 0.75
+	
+	ai_will_do = { has_technology_researched = nationalism }
+
+	possible = {
+		AND = {
+			has_technology_researched = pan-nationalism
+			custom_tooltip = {
+				text = je_german_unification_idea_trigger_desc
+				has_variable = je_german_unification_idea
+			}		
+		}		
+		custom_tooltip = {
+			text = no_confederation_of_the_rhine_tt
+			NOT = {
+				exists = c:RHN
+			}
+		}	
+	}
+}
+
+ITA = {
+	use_culture_states = yes
+	
+	is_major_formation = yes
+	
+	unification_play = dp_unify_italy
+	leadership_play = dp_leadership_italy
+
+	required_states_fraction = 0.7
+
+	ai_will_do = { has_technology_researched = pan-nationalism }
+
+	possible = {
+		OR = {
+			country_has_primary_culture = cu:north_italian
+			country_has_primary_culture = cu:south_italian
+		}
+		any_country = {
+			OR = {
+					country_has_primary_culture = cu:north_italian
+					country_has_primary_culture = cu:south_italian				
+			}
+			has_technology_researched = pan-nationalism	
```

Comment:
So.. Now any italian country just needs pan-nationalism, instead of all italians having nationalism?
I agree with the pan-nationalism change, but.. It feels off.

Reply:
You think we should make it a percentage too? I can work with that, was only concerned that the ai might lag on tech and prevent nation formation as a result